task/secure/README.mkdn

```diff
@@ -0,0 +1,5 @@
+# Task Template
```

Comment:
Remove or fill out please.

Reply:
This is where I should put all the documentation about our TZ interface :)